tests/e2e/test_rosacli_version.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/pkg/info"
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Get CLI version",
+	labels.Day1,
+	labels.FeatureVersion,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient *rosacli.Client
+		)
+
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+		})
+
+		It("can get the version of rosa CLI while logged out - [id:73743]",
+			labels.Medium,
+			func() {
+				By("Make sure to log out")
+				_, logout_err := rosaClient.Runner.Cmd("logout").Run()
+				Expect(logout_err).To(BeNil())
+				By("Get the version output")
+				version_stdout, version_err := rosaClient.Runner.Cmd("version").Run()
+				Expect(version_err).To(BeNil())
+				stdout, e := version_stdout.ReadString('\n')
+				Expect(e).To(BeNil())
+				By("Check the version output")
+				Expect(stdout).NotTo(BeEmpty())
```

Comment:
Does this line provide any extra coverage that `Expect(stdout).To(ContainSubstring(info.Version))` doesn't already ensure?

Reply:
I suppose not. I'll go ahead and remove it